### PR TITLE
wm4-jaspreet_kaur-full-stack-project-Assessment

### PR DESCRIPTION
**Pull request by jaspreet338**

### CODE REVIEW

**Comment by Dedekind561** [comment]
## Required Features checklist - this is for me to check the requirements @jaspreet338 

- [x] 1. Videos should be loaded from a local javascript variable containing the data included in `exampleresponse.json`
- [x] 2. For each video, display a React component that contains
- [x]    - The videos title
- [x]    - An embedded video
- [x]    - The number of votes the video has
- [x]    - A button that when clicked removes the video
- [x] 3. On each video submission there should be two buttons
- [x]    - "Up Vote" - This increases the vote score when clicked
- [x]    - "Down Vote" - This decreases the vote score when clicked
- [x] 4. On the page there must be another React component that will add a Video.
- [x]    - It should include fields to add a
- [x]      - Title
- [x]      - Url
- [x]    - When a button is clicked the video should be added to the list
- [ ] 5. Your website must follow accessibility guidelines (see below for more details)

**Review by Dedekind561 — CHANGES_REQUESTED** [review]
Hello there @jaspreet338, 
Great job so far! I can see some great effort has gone into the work in this PR - so well done!
Just a few things re the requirements. You need to double check the [accessibility guidelines](https://github.com/jaspreet338/Full-Stack-Project-Assessment/blob/jaspreet_kaur/100.md#accessible-guidelines) in the `100.md` file - I think you're missing some things from there.

Other than this you met all the requirements for this level and your code is looking good.  I've left a few more comments re your variable names etc so make sure you check these out. 😄

File: client/src/AddVideo.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
+
+const AddVideo = ({newData}) =>{
+    const [title, setTitle] = useState("");
+    const[url, setUrl] = useState("");
+
+    const addVid = () => {
+        newData({title, url})
```
[the remark this answers is not in the document]
Comment (jaspreet338) [reply client/src/AddVideo.js:8]: thanks @Dedekind561 for your suggestion, i just used it add new video i will change it as per your suggestion

File: client/src/App.js [hunk 2]
```diff
@@ -1,12 +1,27 @@
+import React, { useState } from "react";
 import "./App.css";
+import Video from "./Video";
+import dataVideos from "./exampleresponse.json";
+import AddVideo from "./AddVideo";
+
 
 function App() {
+  const [del, setDel] = useState(dataVideos);
```
[the remark this answers is not in the document]
Comment (jaspreet338) [reply client/src/App.js:9]: delete

File: client/src/AddVideo.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    }
+    return (
+      <div>
+        <label for="title">Title:</label>
+        <input
+          type="text"
+          id="title"
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
```
[the remark this answers is not in the document]
Comment (jaspreet338) [reply client/src/AddVideo.js:17]: thank you

File: client/src/App.js [hunk 4]
```diff
@@ -1,12 +1,27 @@
...
+  const remove = (id) => {
+    let data = del.filter(x => x.id !== id);
```
[the remark this answers is not in the document]
Comment (jaspreet338) [reply client/src/App.js:11]: ok i will change it, thanks for your suggestion

File: client/src/EmbedVideo.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+function EmbedVideo({video}) {
+const url = video.url.replace("watch?v=", "embed/");
+ //console.log(url);
```
[the remark this answers is not in the document]
Comment (jaspreet338) [reply client/src/EmbedVideo.js:5]: i will be more carefull next time

**Pull request closed without merging**